docs/source/CHANGELOG.rst

```diff
@@ -25,13 +29,23 @@ Compatibility
 Bug Fixes
 ~~~~~~~~~~~~~~~~~~
 
+<<<<<<< HEAD
```

Comment:
Same here. It looks like a git conflict.

Reply:
Done.